arduino/cores/packagemanager/identify.go

```diff
@@ -41,6 +41,7 @@ func (pm *PackageManager) IdentifyBoard(idProps *properties.Map) []*cores.Board
 		}
 		return false, true
 	}
+	// FIXME: TODO...
```

Comment:
I'd remove this, it scares me. :sweat_smile:

Reply:
LOL, ok done